packages/react-widgets/src/scss/widget.scss

```diff
@@ -58,6 +58,10 @@
   justify-content: $picker-btn-justify-content;
   padding-left: $picker-btn-padding-left;
   padding-right: $picker-btn-padding-right;
+
+  &:not(:last-child) {
```

Comment:
not loving the way this looks... figured it was better than creating different CSS classes for all different possible combinations of buttons, but would be up for a better idea

Reply:
I would add a dropdown specific class for the wrapping div (to remove the inline flex style), you could handle this there.